src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -628,9 +628,10 @@
       },
       setFocus() {
         this.$nextTick(()=> {
-          this.$refs.amountcredit.focus();
-          this.$refs.amountcredit.select();
-
+	  if(this.$refs.amountcredit) {
+            this.$refs.amountcredit.focus();
+            this.$refs.amountcredit.select();
+	  }
```

Comment:
Allright, can you fix the indentation please ? (mixed spaces and tabs). Thanks ahead.

Reply:
@SeddikKadi There still is a problem with line 634, which contains a tab.

Reply:
@SeddikKadi can you remove the tab at the beginning of line 634 please ?

And also rebase on `main` ? Thanks ahead.